pkg/container/resolver_test.go

```diff
@@ -133,25 +131,18 @@ func TestResolverLocalManifest(t *testing.T) {
 		t.Skip("Podman not available, skipping test")
 	}
 
-	containerFile, err := os.CreateTemp("", "Containerfile")
-	if err != nil {
-		t.Fatalf("Failed to create tempfile: %v", err)
-	}
-	defer os.Remove(containerFile.Name())
+	containerFile, err := os.CreateTemp(t.TempDir(), "Containerfile")
+	assert.NoError(t, err)
 
+	// we can't use `t.TempDir()` here because the cleanup
+	// fails when trying to remove the `<tmpstorage>/overlay`
+	// subdirectory
 	tmpStorage, err := os.MkdirTemp("", "storage")
```

Comment:
It wasn't possible to do `t.TempDir` here because it fails to cleanup the `overlay` subdirectory, so I added a note :)

Reply:
Thanks.  I was wondering about that.